base/files/important_file_writer_unittest.cc

```diff
@@ -367,6 +367,7 @@ TEST_F(ImportantFileWriterTest, DoScheduledWrite_FailToSerialize) {
   histogram_tester.ExpectTotalCount("ImportantFile.WriteDuration", 0);
 }
 
+#if !defined(STARBOARD)
```

Comment:
Are the disabled tests inherently incompatible with cobalt or do we want to enable them later on?